bundle.yaml

```diff
@@ -99,8 +99,8 @@ spec:
         - --kubelet-service=kube-system/kubelet
         - -logtostderr=true
         - --config-reloader-image=quay.io/coreos/configmap-reload:v0.0.1
-        - --prometheus-config-reloader=quay.io/coreos/prometheus-config-reloader:v0.22.2
-        image: quay.io/coreos/prometheus-operator:v0.22.2
+        - --prometheus-config-reloader=quay.io/coreos/prometheus-config-reloader:v0.22.1
```

Comment:
these should be 0.23.0

Reply:
funny, this was the result of running make generate. make generate-in-docker produces very different results.